winterfell/services/authService.js

```diff
@@ -1,21 +1,23 @@
 var _ = require('lodash');
+var bcrypt = require('bcryptjs');
 var SocialUser = require('../utils/socialUser');
 var User = require('../models/user');
 var UserValues = require('../models/userValues');
 var UserService = require('../services/userService');
 
+var salt = bcrypt.genSaltSync(10);
```

Comment:
Each user should have their own salt, otherwise it defeats the purpose of the salt, cryptographically speaking